content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +103,89 @@ For example:
      "type": "MODIFIED",
      "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
    }
-   ...
+   …
    ```
 
-A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
+   {{< note >}}
+   Although the API server responded with a `Content-Type` header indicating a JSON response,
+   the response to a _watch_ request is in fact a stream of multiple documents. For JSON format, the client can
+   detect the end of each JSON document and begin parting the next one.
+   {{< /note >}}
+
+A given Kubernetes cluster only retains a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default. If a requested Watch operation fails because the historical version of that resource is not available, clients must handle the case by recognizing the HTTP response status 410 (Gone). When a client sees a 410 response it should clear any relevant records from its local cache, perform a new List operation, and request a new Watch that starts from the `resourceVersion` returned by that new List operation.
+
```

Comment:
Questions about formatting the response code as well as list and watch?

When a client receives a `410` response, it must clear any relevant records from its local cache, perform a new `list` operation, and request a new `watch` that starts from the `resourceVersion` returned by that new `list` operation.

Reply:
I've omitted this change from the latest revision.